cpp/tests/streams/labeling_bins_test.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ * 
```

Comment:
Trim trailing whitespace...
```suggestion
 *
```